keyboards/ibm/model_m/yacobo/yacobo.c

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2024 Stephen Edwards
+ *
+ * Adapted from the Model H Project by Jonathan Hawthorn.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+
+void keyboard_pre_init_kb(void) {
+    gpio_set_pin_output(C13);
+    gpio_write_pin(C13, 0);
```

Comment:
What's this for?

Reply:
This is the status LED on the Blue Pill. I'll move it to a constant in `config.h`

Reply:
Ah no need -- the `config.h` file can be removed and the pin name simply put in the function call. Just asking for clarification.